rotkehlchen/externalapis/etherscan.py

```diff
@@ -451,17 +463,34 @@ def get_token_transaction_hashes(
             to_block = self.get_blocknumber_by_time(ts=to_ts, closest='before')
             options['endBlock'] = str(to_block)
 
-        hashes: set[tuple[str, Timestamp]] = set()
+        hashes: set[tuple[EVMTxHash, Timestamp]] = set()
         while True:
             result = self._query(module='account', action='tokentx', options=options)
             last_ts = deserialize_timestamp(result[0]['timeStamp']) if len(result) != 0 else None
             for entry in result:
-                timestamp = deserialize_timestamp(entry['timeStamp'])
+                try:
+                    timestamp = deserialize_timestamp(entry['timeStamp'])
+                except DeserializationError as e:
+                    self.msg_aggregator.add_error(f"Failed to query information for transaction {entry['hash']}. Check the logs for more information")  # noqa: E501
+                    log.error(
+                        f"Failed to read transaction timestamp {entry['hash']} from etherscan for "
+                        f'{account} in the range {from_ts} to {to_ts}. {e!s}',
+                    )
+                    continue
+
                 if timestamp > last_ts and len(hashes) >= TRANSACTIONS_BATCH_NUM:  # type: ignore
                     yield _hashes_tuple_to_list(hashes)
                     hashes = set()
                     last_ts = timestamp
-                hashes.add((entry['hash'], timestamp))
+                try:
+                    hashes.add((deserialize_evm_tx_hash(entry['hash']), timestamp))
+                except DeserializationError as e:
+                    self.msg_aggregator.add_error(f"Failed to query information for transaction {entry['hash']}. Check the logs for more information")  # noqa: E501
```

Comment:
Same here. If we wanted to notify the user it should be a different type of websocket message that can be aggregated by the frontend.